__tests__/app.test.js

```diff
@@ -254,17 +257,47 @@ describe("POST /api/articles/:article_id/comments", () => {
         expect(body).toEqual({ msg: "Missing required fields" });
       });
   });
-  // test("400: Responds with a bad request when incorrect data type is passed in the obj", () => {
-  //   const newComment = {
-  //     username: 1,
-  //     body: 100,
-  //   };
-  //   return request(app)
-  //     .post("/api/articles/3/comments")
-  //     .send(newComment)
-  //     .expect(400)
-  //     .then(({ body }) => {
-  //       expect(body).toEqual({ msg: "Bad Request" });
-  //     });
-  // });
+
+  test("400: Responds with a bad request when incorrect data type is passed in the obj", () => {
+    const newComment = {
+      username: 1,
+      body: 100,
+    };
+    return request(app)
```

Comment:
- [ ] TEST -  400 missing required field on post object

Reply:
Isn't this same as the one on line 250?